### ISSUE TITLE
Request Feature Keyword

### ISSUE THREAD

**Issue by alfaris321-coder** [artiebits/svelte-seo#6]
Hi, Thanks for your awesome library, this very2 helpful,  but i woul to like request property keyword for this ..

can we make pull request for this  ?

**Comment by artiebits** [comment]
Hi @alfaris321-coder thanks for the suggestion 👍  
Let me know if you want to make the pull request, or I can do it, no problem

**Comment by artiebits** [comment]
Hi @alfaris321-coder 
I have published a beta version `1.0.2-beta` with keywords, so you can you it like this
```
<SvelteSeo
  title="Article Page Title"
  description="Description of article page"
  keywords="Keywords of article page"
  openGraph={} />
```
Please let me know if it works as you expected

**Comment by artiebits** [comment]
I have released version 1.0.2 with `keywords`. hope it solves your issue
